src/DependencyInjection/Loader/ConfigurableCallValuesCollectingPhpFileLoader.php

```diff
@@ -36,9 +36,9 @@ public function load($resource, $type = null): void
 
     public function import(
         $resource,
-        $type = null,
+        string $type = null,
         $ignoreErrors = false,
-        $sourceResource = null,
+        string $sourceResource = null,
```

Comment:
this update is required as new symfony/dependency-injection release which update the param type